applications/acdc/src/acdc_agent_listener.erl

```diff
@@ -1032,6 +1033,7 @@ maybe_connect_to_agent(MyQ, EPs, Call, Timeout, AgentId, _CdrUrl) ->
                                   ,{<<"Retain-CID">>, <<"true">>}
                                   ,{<<"Agent-ID">>, AgentId}
                                   ,{<<"Member-Call-ID">>, MCallId}
+                                  ,{<<"originate_signal_bond">>, MCallId}
```

Comment:
I can't speak to the ACDc-specific portions, but this name should match the format of other keys: `Originate-Signal-Bond` in this case.